### ISSUE TITLE
Some questions about Dataset Setup

### ISSUE THREAD

**Issue by votuongquan** [teamtee/Qwen2-Audio-finetune#4]
Dear teamtee,
Thank you for creating and sharing the Qwen2-Audio-finetune repository—it's a great resource for anyone looking to finetune Qwen2-Audio models.

I'm trying to prepare my own dataset (a Vietnamese dataset called VIVOS, you can have a look over it [at this link](https://www.kaggle.com/datasets/kynthesis/vivos-vietnamese-speech-corpus-for-asr)) for finetuning, but I'd appreciate some more guidance. Could you please explain in more detail how to prepare the dataset, including the specific formats and structures of data sources, the my_wav.scp file,  and how can I generate or obtain the data from only wav files and text transcription files?
Additionally, If I have separate train and test splits (as you can see in the dataset link above), should I use "train split" for both train and eval or "train for train and test for eval", and my multitask.jsonl contains both sources or should i split it.

Your insights would be incredibly helpful. Thank you in advance for your time!
Best regards,
Quan Vo TUong

**Comment by teamtee** [comment]
Hello, regarding how to prepare the my_wav.scp and multitask.jsonl files, you can do so by writing a simple Python script, or seek help from tools like ChatGPT—you just need to ensure the format and content of the files are correct.  As for the training process, we usually split the dataset into three parts: train, eval, and test. Specifically, the train (training set) and eval (validation set) should be used during training, while the test (test set) is reserved for evaluation.

**Comment by teamtee** [comment]
You can split 5% of the training set to use as the eval set.

**Comment by votuongquan** [comment]
Sorry, could you help me answer my question? I now have the code to generate data, but should I split the data folder into train and eval folders with different data_wav.1.ark and data_wav.1.scp files, or should I keep a single folder that contains all the information for both? And the multitask.jsonl and my_wav.scp contain both data, right? If possible, could you show me your real structure of data when you run the code? Thank you very much
